build.gradle

```diff
@@ -18,7 +18,8 @@ buildscript {
 }
 
 plugins {
-    id 'org.jetbrains.intellij' version '1.15.0'
+    id 'org.jetbrains.kotlin.jvm' version '1.9.10'
```

Comment:
why do we need kotlin?

Reply:
from GH action that check next EAP ( https://github.com/redhat-developer/intellij-openshift-connector/actions/workflows/IJ-latest.yml ) there was errors about kotlin : 
`java.lang.NoSuchMethodError: 'kotlin.sequences.Sequence kotlin.io.path.PathsKt.walk(java.nio.file.Path, kotlin.io.path.PathWalkOption[])'`
that made almost ALL tests to fails.
using that plugin made those tests passes again

Reply:
however, looking at the differents validation builds, there is some messages i would like to check before merging this PR
```
> Task :verifyPluginConfiguration
[gradle-intellij-plugin :verifyPluginConfiguration] The following plugin configuration issues were found:
- The 'since-build' property is lower than the target IntelliJ Platform major version: 212 < 232.
- The Java configuration specifies targetCompatibility=17 but since-build='212' property requires targetCompatibility=11.
- The Kotlin configuration specifies jvmTarget=17 but since-build='212' property requires jvmTarget=11.
- The dependency on the Kotlin Standard Library (stdlib) is automatically added when using the Gradle Kotlin plugin and may conflict with the version provided with the IntelliJ Platform, see: https://jb.gg/intellij-platform-kotlin-stdlib
See: https://jb.gg/intellij-platform-versions
```

Reply:
removed this dependency